src/YotiClient.php

```diff
@@ -61,9 +60,6 @@ public function __construct(
         Validation::notEmptyString($sdkId, 'SDK ID');
         $pemFile = PemFile::resolveFromString($pem);
 
-        // Create default HTTP client.
-        $options[Config::HTTP_CLIENT] = $options[Config::HTTP_CLIENT] ?? new Client();
```

Comment:
`Config::getHttpClient()` has been changed to return a default client when one hasn't been configured, so setting here is no longer needed